website/content/docs/sync/index.mdx

```diff
@@ -127,6 +127,24 @@ Granularity can be updated. The new granularity only affects secrets newly assoc
 not modify the previously synced secrets. It is possible to update an association to force a recreate operation.
 The secret synced with the old granularity will be deleted and new secrets will be synced according to the new granularity.
 
+
+## Reconciliation
+
+Vault Secrets Sync is designed to automatically recover from transient failures
+in two ways: operation retries and reconciliation scans.
+
+Operation retries happen when a sync operation fails. Vault automatically
+retries the operation with exponential backoff. Operation retries help in
+situations where your network becomes unreliable or overwhelmed.
+
+Reconciliation scans happen periodically in a background thread. Vault scans all
```

Comment:
Should we say how often or this is too detailed and subject to change?

Reply:
Personally I think it's too detailed and subject to change, especially once we have a dedicated configuration for it.